backend/tasks/username.py

```diff
@@ -0,0 +1,31 @@
+from flask import request
+from flask_restful import Resource
+from flask import make_response
+from flask_restful import reqparse
+import traceback
+
+class UsernameTask(Resource):    
+    def put(self):
+        parser = reqparse.RequestParser()
+        parser.add_argument('newUsername', type=str, help='new username')
+        parser.add_argument('currentUsername', type=str, help='current username')
+        args = parser.parse_args()
+        new_user_name = args.get("newUsername")
```

Comment:
We should do some validation that the username only contains contains alphanumerical characters and only starts with a letter.